src/main/java/com/coveros/selenified/element/check/Equals.java

```diff
@@ -268,25 +268,25 @@ default String checkText(String expectedText, double waitFor, double timeTook) {
      * @return String: the actual text of the table cell. null will be returned if the element isn't present or a table, or table cell doesn't exist
      */
     @SuppressWarnings("squid:S1168")
-    default String checkText(int row, int col, String expectedText, double waitFor, double timeTook) {
+    String checkText(int row, int col, String expectedText, double waitFor, double timeTook) {
         String column = " and column ";
         String within = " within element ";
         // record the action, and check for table
-        if (!isPresentTable("cell at row " + row + column + col + within + getElement().prettyOutput() +
+        if (!isPresentTable("cell at row " + row + column + col + within + this.element.prettyOutput() +
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "cell at row " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)